packages/tables/src/Table/Concerns/CanSearchRecords.php

```diff
@@ -77,4 +79,16 @@ public function getColumnSearchIndicators(): array
     {
         return $this->getLivewire()->getTableColumnSearchIndicators();
     }
+
+    public function searchPlaceholder(string | Closure $searchPlaceholder): static
+    {
+        $this->searchPlaceholder = $searchPlaceholder;
+
+        return $this;
+    }
+
+    public function getSearchPlaceholder(): ?string
+    {
+        return $this->evaluate($this->searchPlaceholder);
+    }
```

Comment:
Could you move this up after `persistsColumnSearchesInSession`, @bernhardh?

Reply:
done